TypeScript.ContractGenerator/TypeScriptUnit.cs

```diff
@@ -15,7 +15,7 @@ public class TypeScriptUnit
 
         public List<TypeScriptStatement> Body { get; } = new List<TypeScriptStatement>();
 
-        public TypeScriptTypeReference AddTypeImport(ITypeInfo sourceType, TypeScriptTypeDeclaration typeDeclaration, TypeScriptUnit sourceUnit)
+        public TypeScriptTypeReference AddTypeImport(ITypeInfo sourceType, TypeScriptTypeDeclaration typeDeclaration, TypeScriptUnit sourceUnit, bool useTypeKeyword=false)
```

Comment:
let's remove default parameter for useTypeKeyword, always pass parameter explicitly